assets/src/application/ui/input/InlineEditInput/types.ts

```diff
@@ -1,8 +1,13 @@
 import { InlineEditProps } from '@infraUI/inputs';
 
-export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> {}
+export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> { }
 
 export interface TextProps extends Omit<InlineEditProps, 'inputType'> {
 	fitText?: boolean;
 	tag?: React.ElementType;
 }
+
+export interface TabbableTextProps {
+	onClick: VoidFunction;
```

Comment:
`onClick` is too generic. May be name it something very specific - related to what it actually does, like `onEdit`, `onRequestEdit` etc.

Reply:
this is funny, that's what I did in previous commit, but thought it wouldn't be clear why I am passing `onRequestEdit` prop, will update now